daml-lf/spec/daml-lf-1.rst

```diff
@@ -2711,6 +2711,8 @@ ContractId functions
   Returns ``'True'`` if the first contact id is equal to the second,
   ``'False'`` otherwise.
 
+  [*Available in versions < 1.dev*]
```

Comment:
I don’t think we ever discussed this and I still use this in DAML script and DAML triggers. Did you mean to change `EQUAL_CONTRACT_ID` instead of `COERCE_CONTRACT_ID`?

Reply:
My mistake. Thanks to catch that

Reply:
The ``[*Available in version ...*]`` in the built-in section are always at the end of the description of the built. This annotation is about ``EQUAL_CONTRACT_ID`` not ``COERCE_CONTRACT_ID``

Reply:
Oh I see, I got confused by the diff. LGTM  to me then :+1: